LRQA-80963 TF

JoyceWang08211 opened the pull request.

lipusz committed:
LPS-187860 portal-workflow-metrics-service: Use new methods to set mappings and settings

https://issues.liferay.com/browse/LPS-187860

lipusz committed:
LPS-187860 portal-workflow-metrics-service: Separate the mappings of each index into their own file

https://issues.liferay.com/browse/LPS-187860

lipusz committed:
LPS-187860 portal-search-engine-adapter-api: baseline

https://issues.liferay.com/browse/LPS-187860

xbrianlee committed:
LPS-181205 Add test to configure Search Bar to show Top and Trending searches leveraging data from Analytics Cloud

joshuacords committed:
LPS-187860 portal-search-tuning-rankings-web: Update test

https://issues.liferay.com/browse/LPS-187860

joshuacords committed:
LPS-187860 portal-search-tuning-synonyms-web: Update test

https://issues.liferay.com/browse/LPS-187860

markocikos committed:
LPS-186438 Set `label` as title field, shown in fragment configuration

pedro-oliveira446 committed:
LPS-184962 Add localized fields in object's layout

pedro-oliveira446 committed:
LPS-184962 Extract common parts to util directory to reuse in new component

pedro-oliveira446 committed:
LPS-184962 Make onchange change all locales dropdowns

pedro-oliveira446 committed:
LPS-184962 Change AvailableLocaleLabel file to typescript

pedro-oliveira446 committed:
LPS-184962 Change LocalesDropdown file to typescript

fmarcio committed:
LRAC-13729 create lang

fmarcio committed:
LRAC-13729 update .catch on promise to display warning message

fortunatomaldonado committed:
LPS-186633 Scale image by adding height attribute

thiagobuarque-liferay committed:
LRAC-13729 update request with messageKey if statement

pedro-oliveira446 committed:
LPS-184962 Add parameters to rich text field accept localized values

pedro-oliveira446 committed:
LPS-184962 Adapt Rich Text to accept localized values in frontend

ericthmoritsuka committed:
LPS-167460 added margin bottom to options button and adjusted toggle function

pedro-oliveira446 committed:
LPS-184962 SF

pedro-oliveira446 committed:
LPS-184962 Fix frontend snapshot tests

pedro-oliveira446 committed:
LPS-184962 Semver

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

JoyceWang08211 commented:
ci:test:security:nocompile

liferay-continuous-integration commented:
The test will run with a prebuilt bundle.

liferay-continuous-integration commented:
<html><h3>:x: ci:test:security - 0 out of 3 jobs passed in 53 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4603">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1347401_4212</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/6bf7e58d233a3f52beb4a2374904ef7f1819ad36">6bf7e58d233a3f52beb4a2374904ef7f1819ad36</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/0c55dacee2cdc8567029272f565884396e88974e">0c55dacee2cdc8567029272f565884396e88974e</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2347308985">EE Development Acceptance (master) - 4602 - 2023-06-11[12:34:37]</a></p><details><summary><strong>ci:test:security - 0 out of 3 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/4212/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/232190/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/213349/"><strike>functional-tomcat90-mysql57-jdk8/0/1</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4212/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/232190//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/232190//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId_">LocalFile.ContentSecurityPolicy#VerifyCSPCouldConfiguredByWebId</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-34/test-portal-acceptance-pullrequest(master)/4212/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-34/test-portal-acceptance-pullrequest(master)/4212/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-34/test-portal-acceptance-pullrequest(master)/4212/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/213349//consoleText">functional-tomcat90-mysql57-jdk8/0/1</a><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/213349//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource_">LocalFile.ContentSecurityPolicy#VerifyCSPCouldConfiguredToBlockImageSource</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-34/test-portal-acceptance-pullrequest(master)/4212/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-34/test-portal-acceptance-pullrequest(master)/4212/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-34/test-portal-acceptance-pullrequest(master)/4212/functional-tomcat90-mysql57-jdk8/0/1/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/4212/">test-portal-acceptance-pullrequest(master)#4212</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4212/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>security<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/182">JoyceWang08211#182</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2347710754">[master] ci:test:security - JoyceWang08211 &gt; JoyceWang08211 - PR#182 - 2023-06-12[01:27:18]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-34.liferay.com/job/publish-testray-report/9779/">publish-testray-report#9779</a><br/></div></div>

michaelhashimoto committed:
LRCI-3544 Add the Message JSONObject into the Event Handler constructor

michaelhashimoto committed:
LRCI-3544 Add event handler for Starting & Completing builds

michaelhashimoto committed:
LRCI-3544 Add BaseObjectEventHandler to distinguish from BaseJenkinsEventHandlers

michaelhashimoto committed:
LRCI-3544 Add methods to the BaseEventHandler to get repository & queue objects

ling-alan-huang committed:
LPS-185667 SF, avoid stale object

ling-alan-huang committed:
LPS-185667 SF, avoid stale object

ling-alan-huang committed:
LPS-185667 SF, avoid stale object

ling-alan-huang committed:
LPS-185667 SF, avoid stale object

ling-alan-huang committed:
LPS-185667 SF, avoid stale object

michaelhashimoto committed:
LRCI-3544 Add additional Event Types to the Event Handler

michaelhashimoto committed:
LRCI-3544 Relate the JMSEventHandler to the EventHandlerHelper

michaelhashimoto committed:
LRCI-3544 Update the BuildParameter value to a LongText

michaelhashimoto committed:
LRCI-3544 SF

michaelhashimoto committed:
LRCI-3544 Add Jenkins Computer Event Handlers to handle computer statuses

michaelhashimoto committed:
LRCI-3544 Add interactions to handle JMS & REST API endpoints

michaelhashimoto committed:
LRCI-3544 Always update the build queue & jenkins queue to invoke on create build events

ces-locpham committed:
LPS-187590 Fix startDate and endDate auto change to the default system timezone date

oliverkecskemety committed:
LPS-176414 Switch intervals to cron expressions

oliverkecskemety committed:
LPS-176414 Autogenerate language keys (gw buildLang)

victorg1991 committed:
LPS-187597 Not needed

victorg1991 committed:
LPS-187597 Make sure it is displayed after configure page icon

victorg1991 committed:
LPS-187597 Convert link to button to improve accessibility

oliverkecskemety committed:
LPS-176414 Baseline

carolmariaabb committed:
LPS-187634 Change from ObjectEntryService to ObjectEntryManager to be able to use Proxy Objects in form container

Commit:
LPS-187661 Expose the issue in the PortalInstances.initCompany().

Commit:
LPS-187661 PortalInstances.initCompany() doesn't reset the new webId to the existed key after updating the company's webId.

Commit:
LPS-187661 SF, use @DeleteAfterTestRun.

victorg1991 committed:
LPS-187978 Don't throw null

BryanEngler committed:
LPS-187860 commerce-machine-learning-impl: Use new methods to set mappings and settings

https://issues.liferay.com/browse/LPS-187860

BryanEngler committed:
LPS-187860 portal-search-elasticsearch7-impl: Update request executor

https://issues.liferay.com/browse/LPS-187860

BryanEngler committed:
LPS-187860 portal-search-engine-adapter-api: Add methods to set mappings and settings separately when creating an index

https://issues.liferay.com/browse/LPS-187860

BryanEngler committed:
LPS-187860 portal-search-tuning-rankings-web: Split index.json into mappings.json and settings.json

https://issues.liferay.com/browse/LPS-187860

BryanEngler committed:
LPS-187860 portal-search-tuning-rankings-web: Use new methods to set mappings and settings

https://issues.liferay.com/browse/LPS-187860

BryanEngler committed:
LPS-187860 portal-search-tuning-synonyms-web: Split index.json into mappings.json and settings.json

https://issues.liferay.com/browse/LPS-187860

BryanEngler committed:
LPS-187860 portal-search-tuning-synonyms-web: Use new methods to set mappings and settings

https://issues.liferay.com/browse/LPS-187860

BryanEngler committed:
LPS-187860 commerce-machine-learning-impl: Try to standardize the *-mappings.json pattern

https://issues.liferay.com/browse/LPS-187860

michaelhashimoto committed:
LRCI-3544 Fix after rebase

BryanEngler committed:
LPS-187860 commerce-machine-learning-impl: Rename files to follow pattern

https://issues.liferay.com/browse/LPS-187860

marcosapmf committed:
LRAC-14056 re-enable wedeploy tasks

marcosapmf committed:
LRAC-14056 update docker image version

marcosapmf committed:
LRAC-14056 use rewrite.config

marcosapmf committed:
LRAC-14056 update context to be able to use rewrite.config

marcosapmf committed:
LRAC-14056 apply

Commit:
COMMERCE-10889 add new supplier account role

Commit:
COMMERCE-10889 when account supplier role is applied also regular supplier is

Commit:
COMMERCE-10889 hide supplier role if not a supplier account

marcosapmf committed:
LRAC-13967 add btn classes

marcosapmf committed:
LRAC-13967 Regen Snapshots

tinatian committed:
LPS-187661 Retrieve companyId from theme display as PropsValues.COMPANY_DEFAULT_WEB_ID is changed but webIds stored in PortalInstances not updated yet.

pyoo47 committed:
LRCI-3640 Console text loaders should always bypass the console file size limit for top-level builds.

david-truong committed:
LPS-186466 using deleteRow allows us to combine the constraint resolver

liulukiwi committed:
LPS-187597 Update toggle controls locator

istvansajtos committed:
LPS-187515 Rejected requests due to missing headers

Commit:
COMMERCE-10889 baseline

jkappler committed:
LPS-187933 Adds throwable message to the log

danielCamara02 committed:
LPS-187676 Fix AssertFederatedSearchRequestsInheritParentRequestsSearchContextProperties test

marcosapmf committed:
LRAC-14056 Sort

marcosapmf committed:
LRAC-14056 SF

oliverkecskemety committed:
LPS-176414 Whitelist scheduler classes

Commit:
LPS-179293 Refactor tests that to setSEOcanonicalURL inside Echo repo

ealonso committed:
LPS-187978 If urlViewInContext is null, return it using _assetDisplayPageFriendlyURLProvider

4lejandrito committed:
LPS-187966 This should have been true from the start

It does not make a difference though because the modifiable property is
being ignored in ObjectDefinitionResourceImpl#postObjectDefinition. It
was already been set to true.

brianchandotcom committed:
artifact:ignore depot-web 2.0.82 change log

brianchandotcom committed:
artifact:ignore depot-web 2.0.82 prep next

brianchandotcom committed:
artifact:ignore depot-web 2.0.82 artifact properties

brianchandotcom committed:
artifact:ignore document-library-item-selector-web 5.0.47 change log

brianchandotcom committed:
artifact:ignore document-library-item-selector-web 5.0.47 prep next

brianchandotcom committed:
artifact:ignore document-library-item-selector-web 5.0.47 artifact properties

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-field-type 6.0.145 change log

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-field-type 6.0.145 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-field-type 6.0.145 artifact properties

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-web 4.0.144 change log

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-web 4.0.144 artifact properties

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-web 4.0.144 prep next

brianchandotcom committed:
artifact:ignore export-import-web 5.0.70 change log

JLucasM committed:
LRAC-14037 Change DropDown component to fix the link

brianchandotcom committed:
artifact:ignore export-import-web 5.0.70 prep next

brianchandotcom committed:
artifact:ignore export-import-web 5.0.70 artifact properties

brianchandotcom committed:
artifact:ignore fragment-item-selector-web 1.0.32 change log

brianchandotcom committed:
artifact:ignore fragment-item-selector-web 1.0.32 prep next

brianchandotcom committed:
artifact:ignore fragment-item-selector-web 1.0.32 artifact properties

brianchandotcom committed:
artifact:ignore fragment-web 4.0.86 change log

brianchandotcom committed:
artifact:ignore fragment-web 4.0.86 artifact properties

brianchandotcom committed:
artifact:ignore fragment-web 4.0.86 prep next

brianchandotcom committed:
artifact:ignore journal-item-selector-web 5.0.16 change log

brianchandotcom committed:
artifact:ignore journal-item-selector-web 5.0.16 prep next

brianchandotcom committed:
artifact:ignore journal-item-selector-web 5.0.16 artifact properties

brianchandotcom committed:
artifact:ignore knowledge-base-item-selector-web 5.0.17 change log

brianchandotcom committed:
artifact:ignore knowledge-base-item-selector-web 5.0.17 prep next

brianchandotcom committed:
artifact:ignore knowledge-base-item-selector-web 5.0.17 artifact properties

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.169 change log

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.169 prep next

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.169 artifact properties

brianchandotcom committed:
artifact:ignore notification-web 1.0.54 artifact properties

brianchandotcom committed:
artifact:ignore notification-web 1.0.54 prep next

brianchandotcom committed:
artifact:ignore object-admin-rest-impl 1.0.94 change log

brianchandotcom committed:
artifact:ignore object-admin-rest-impl 1.0.94 prep next

brianchandotcom committed:
artifact:ignore object-admin-rest-impl 1.0.94 artifact properties

brianchandotcom committed:
artifact:ignore object-rest-impl 1.0.115 change log

brianchandotcom committed:
artifact:ignore object-rest-impl 1.0.115 artifact properties

brianchandotcom committed:
artifact:ignore object-rest-impl 1.0.115 prep next

brianchandotcom committed:
artifact:ignore object-service 1.0.157 change log

brianchandotcom committed:
artifact:ignore object-service 1.0.157 prep next

brianchandotcom committed:
artifact:ignore object-service 1.0.157 artifact properties

brianchandotcom committed:
artifact:ignore object-web 1.0.171 change log

brianchandotcom committed:
artifact:ignore object-web 1.0.171 prep next

brianchandotcom committed:
artifact:ignore object-web 1.0.171 artifact properties

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.107 change log

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.107 artifact properties

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.107 prep next

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 6.0.84 change log

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 6.0.84 artifact properties

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 6.0.84 prep next

brianchandotcom committed:
artifact:ignore roles-item-selector-web 4.0.15 change log

brianchandotcom committed:
artifact:ignore roles-item-selector-web 4.0.15 prep next

brianchandotcom committed:
artifact:ignore roles-item-selector-web 4.0.15 artifact properties

brianchandotcom committed:
artifact:ignore site-item-selector-web 6.0.31 change log

brianchandotcom committed:
artifact:ignore site-item-selector-web 6.0.31 prep next

brianchandotcom committed:
artifact:ignore site-item-selector-web 6.0.31 artifact properties

brianchandotcom committed:
artifact:ignore site-navigation-item-selector-web 4.0.36 change log

brianchandotcom committed:
artifact:ignore site-navigation-item-selector-web 4.0.36 prep next

brianchandotcom committed:
artifact:ignore site-navigation-item-selector-web 4.0.36 artifact properties

brianchandotcom committed:
artifact:ignore users-admin-item-selector-web 4.0.16 change log

brianchandotcom committed:
artifact:ignore users-admin-item-selector-web 4.0.16 prep next

brianchandotcom committed:
artifact:ignore users-admin-item-selector-web 4.0.16 artifact properties

brianchandotcom committed:
artifact:ignore users-admin-web 6.0.102 change log

brianchandotcom committed:
artifact:ignore users-admin-web 6.0.102 artifact properties

brianchandotcom committed:
artifact:ignore users-admin-web 6.0.102 prep next

brianchandotcom committed:
artifact:ignore wiki-web 7.0.91 change log

brianchandotcom committed:
artifact:ignore wiki-web 7.0.91 prep next

brianchandotcom committed:
artifact:ignore wiki-web 7.0.91 artifact properties

brianchandotcom committed:
artifact:ignore headless-builder-impl 1.0.6 change log

brianchandotcom committed:
artifact:ignore headless-builder-impl 1.0.6 artifact properties

brianchandotcom committed:
artifact:ignore headless-builder-impl 1.0.6 prep next

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.94 change log

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.94 prep next

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.94 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.34 change log

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.34 prep next

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.34 artifact properties

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.45 change log

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.45 prep next

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.45 artifact properties

brianchandotcom committed:
build.gradle auto SF

pietrobocale committed:
COMMERCE-11799 Fix macro createAccount

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.78 change log

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.78 artifact properties

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.78 prep next

brianchandotcom committed:
artifact:ignore commerce-shop-by-diagram-web 1.0.67 change log

brianchandotcom committed:
artifact:ignore commerce-shop-by-diagram-web 1.0.67 artifact properties

brianchandotcom committed:
artifact:ignore commerce-shop-by-diagram-web 1.0.67 prep next

brianchandotcom committed:
artifact:ignore multi-factor-authentication-email-otp-web 2.0.26 change log

brianchandotcom committed:
artifact:ignore multi-factor-authentication-email-otp-web 2.0.26 prep next

brianchandotcom committed:
artifact:ignore multi-factor-authentication-email-otp-web 2.0.26 artifact properties

brianchandotcom committed:
artifact:ignore multi-factor-authentication-fido2-web 1.0.28 change log

brianchandotcom committed:
artifact:ignore multi-factor-authentication-fido2-web 1.0.28 prep next

brianchandotcom committed:
artifact:ignore multi-factor-authentication-fido2-web 1.0.28 artifact properties

brianchandotcom committed:
artifact:ignore multi-factor-authentication-ip-address-impl 2.0.14 change log

brianchandotcom committed:
artifact:ignore multi-factor-authentication-ip-address-impl 2.0.14 artifact properties

brianchandotcom committed:
artifact:ignore multi-factor-authentication-ip-address-impl 2.0.14 prep next

brianchandotcom committed:
artifact:ignore multi-factor-authentication-timebased-otp-web 2.0.22 change log

brianchandotcom committed:
artifact:ignore multi-factor-authentication-timebased-otp-web 2.0.22 prep next

brianchandotcom committed:
artifact:ignore multi-factor-authentication-timebased-otp-web 2.0.22 artifact properties

brianchandotcom committed:
artifact:ignore portal-search-similar-results-web 3.0.38 change log

brianchandotcom committed:
artifact:ignore portal-search-similar-results-web 3.0.38 prep next

brianchandotcom committed:
artifact:ignore portal-search-similar-results-web 3.0.38 artifact properties

brianchandotcom committed:
artifact:ignore portal-search-tuning-rankings-web 3.0.62 change log

brianchandotcom committed:
artifact:ignore portal-search-tuning-rankings-web 3.0.62 prep next

brianchandotcom committed:
artifact:ignore portal-search-tuning-rankings-web 3.0.62 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-api 12.0.0 change log

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-api 12.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-api 12.0.0 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-web 3.0.61 change log

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-web 3.0.61 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-web 3.0.61 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.43 change log

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.43 prep next

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.43 artifact properties

brianchandotcom committed:
artifact:ignore segments-asah-connector 3.0.47 change log

brianchandotcom committed:
artifact:ignore segments-asah-connector 3.0.47 prep next

brianchandotcom committed:
artifact:ignore segments-asah-connector 3.0.47 artifact properties

brianchandotcom committed:
artifact:ignore segments-experiment-web 3.0.70 change log

brianchandotcom committed:
artifact:ignore segments-experiment-web 3.0.70 prep next

brianchandotcom committed:
artifact:ignore segments-experiment-web 3.0.70 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.57 change log

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.57 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.57 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-spi 4.0.0 change log

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-spi 4.0.0 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-spi 4.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-service 3.0.57 change log

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-service 3.0.57 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-service 3.0.57 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-demo-data-creator-impl 2.0.10 change log

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-demo-data-creator-impl 2.0.10 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-demo-data-creator-impl 2.0.10 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-impl 3.0.73 change log

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-impl 3.0.73 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-impl 3.0.73 prep next

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
7.4.3.82 GA82 Release Apps: account, adaptive-media, address, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, feature-flag, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream

brianchandotcom committed:
7.4.13 Update 82 Release Apps: account, adaptive-media, address, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, feature-flag, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream, antivirus, enterprise-product-notification, multi-factor-authentication, portal-search-elasticsearch-cross-cluster-replication, portal-search-elasticsearch-monitoring, portal-search-learning-to-rank, portal-search-similar-results, portal-search-tuning, saml, search-experiences

interaminense committed:
LRAC-14037 SF

allen-nagtalon committed:
LRQA-80464 Remove deprecated AntCommand function

rbohl committed:
LPS-176640, workdsmith the breaking change

ling-alan-huang committed:
LPS-174431 Auto SF

shuyangzhou committed:
LPS-187404 Add more context info in the exception message

rbohl committed:
LPS-187594, wordsmith the breaking change

austinchiang committed:
LPS-187928 Remove unnecessary property

shuyangzhou committed:
LPS-187404 Not liferay service and don't require schema version.

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPS-187597 Inline

brianchandotcom committed:
LPS-187597 Sort

brianchandotcom committed:
LRCI-3640 prep next

brianchandotcom committed:
LPS-176414 Wordsmith

brianchandotcom committed:
LPS-176414 Regen

brianchandotcom committed:
LPS-187860 Rename

brianchandotcom committed:
LPS-187860 SF

brianchandotcom committed:
LPS-187860 SF

brianchandotcom committed:
LPS-184962 Keep it private

brianchandotcom committed:
COMMERCE-10889 Sort

brianchandotcom committed:
LPS-187978 SF

brianchandotcom committed:
LRCI-3544 Wordsmith

brianchandotcom committed:
LRAC-14056 SF

JoyceWang08211 commented:
ci:test:security:nocompile

liferay-continuous-integration commented:
The test will run with a prebuilt bundle.

liferay-continuous-integration commented:
<html><h3>:x: ci:test:security - 0 out of 3 jobs passed in 23 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d120a830a2255fa5fcc02f1dbef55709ec4ed5a8">d120a830a2255fa5fcc02f1dbef55709ec4ed5a8</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d120a830a2255fa5fcc02f1dbef55709ec4ed5a8">d120a830a2255fa5fcc02f1dbef55709ec4ed5a8</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2354291803">EE Development Acceptance (master) - 4626 - 2023-06-18[16:34:08]</a></p><details><summary><strong>ci:test:security - 0 out of 3 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/3423/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/575326/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/688090/"><strike>functional-tomcat90-mysql57-jdk8/0/1</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3423/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/575326//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/575326//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId_">LocalFile.ContentSecurityPolicy#VerifyCSPCouldConfiguredByWebId</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3423/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3423/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3423/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/688090//consoleText">functional-tomcat90-mysql57-jdk8/0/1</a><ol><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/688090//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource_">LocalFile.ContentSecurityPolicy#VerifyCSPCouldConfiguredToBlockImageSource</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3423/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3423/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3423/functional-tomcat90-mysql57-jdk8/0/1/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

JoyceWang08211 commented:
ci:test:security:nocompile

liferay-continuous-integration commented:
The test will run with a prebuilt bundle.

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/3423/">test-portal-acceptance-pullrequest(master)#3423</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3423/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>security<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/182">JoyceWang08211#182</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2354363106">[master] ci:test:security - JoyceWang08211 &gt; JoyceWang08211 - PR#182 - 2023-06-19[00:47:12]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/6332/">publish-testray-report#6332</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:security - 0 out of 3 jobs passed in 24 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4627">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1327401_286</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/e5341d48364c62e329d3c551ded58f3900e3c0b5">e5341d48364c62e329d3c551ded58f3900e3c0b5</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d120a830a2255fa5fcc02f1dbef55709ec4ed5a8">d120a830a2255fa5fcc02f1dbef55709ec4ed5a8</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2354291803">EE Development Acceptance (master) - 4626 - 2023-06-18[16:34:08]</a></p><details><summary><strong>ci:test:security - 0 out of 3 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/286/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/688186/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/187124/"><strike>functional-tomcat90-mysql57-jdk8/0/1</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/286/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/688186//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/688186//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId_">LocalFile.ContentSecurityPolicy#VerifyCSPCouldConfiguredByWebId</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-32/test-portal-acceptance-pullrequest(master)/286/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-32/test-portal-acceptance-pullrequest(master)/286/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-32/test-portal-acceptance-pullrequest(master)/286/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/187124//consoleText">functional-tomcat90-mysql57-jdk8/0/1</a><ol><li><div><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/187124//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource_">LocalFile.ContentSecurityPolicy#VerifyCSPCouldConfiguredToBlockImageSource</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-32/test-portal-acceptance-pullrequest(master)/286/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-32/test-portal-acceptance-pullrequest(master)/286/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-32/test-portal-acceptance-pullrequest(master)/286/functional-tomcat90-mysql57-jdk8/0/1/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

JoyceWang08211 committed:
LRQA-80963 TF

JoyceWang08211 commented:
ci:test:security:nocompile

liferay-continuous-integration commented:
The test will run with a prebuilt bundle.

liferay-continuous-integration commented:
<html><h3>:x: ci:test:security - 0 out of 1 jobs passed in 1 minute</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/e5341d48364c62e329d3c551ded58f3900e3c0b5">e5341d48364c62e329d3c551ded58f3900e3c0b5</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/72b0c6bf2c53b0ab12c84f0d77a4417c06b2b7b0">72b0c6bf2c53b0ab12c84f0d77a4417c06b2b7b0</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2351929143">EE Development Acceptance (master) - 4614 - 2023-06-15[00:43:31]</a></p><details><summary><strong>ci:test:security - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/12497/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/12497/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/12497/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/JoyceWang08211/liferay-portal/pull/182">JoyceWang08211/master-qa-80963</a></strong><pre>Unable to rebase master-temp-1687167308271 to master
Rebasing (1/1)
error: could not apply 307e61e2fc8d... LRQA-80963 TF
Resolve all conflicts manually, mark them as resolved with
"git add/rm &lt;conflicted_files&gt;", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply 307e61e2fc8d... LRQA-80963 TF</pre></p></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

JoyceWang08211 commented:
ci:test:security:nocompile

liferay-continuous-integration commented:
The test will run with a prebuilt bundle.

liferay-continuous-integration commented:
<html><h3>:x: ci:test:security - 0 out of 1 jobs passed in 1 minute</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/e5341d48364c62e329d3c551ded58f3900e3c0b5">e5341d48364c62e329d3c551ded58f3900e3c0b5</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/72b0c6bf2c53b0ab12c84f0d77a4417c06b2b7b0">72b0c6bf2c53b0ab12c84f0d77a4417c06b2b7b0</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2351929143">EE Development Acceptance (master) - 4614 - 2023-06-15[00:43:31]</a></p><details><summary><strong>ci:test:security - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/8759/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8759/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/8759/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/JoyceWang08211/liferay-portal/pull/182">JoyceWang08211/master-qa-80963</a></strong><pre>Unable to rebase master-temp-1687167543107 to master
Rebasing (1/1)
error: could not apply 307e61e2fc8d... LRQA-80963 TF
Resolve all conflicts manually, mark them as resolved with
"git add/rm &lt;conflicted_files&gt;", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply 307e61e2fc8d... LRQA-80963 TF</pre></p></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>
